core/src/gl/uniformEntries.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string>
+
+namespace Tangram {
+
+namespace UniformEntries {
+
+typedef uint16_t EntryId;
```

Comment:
Why not `using`?

Also what about a signed type so we could use the sign to signify invalid IDs?